client/packages/common/src/ui/layout/tables/material-react-table/useGetColumnDefDefaults.tsx

```diff
@@ -89,6 +91,17 @@ export const useGetColumnTypeDefaults = () => {
           },
         };
 
+      case ColumnType.Boolean:
+        return {
+          align: 'center',
+          Cell: ({ cell }) => {
+            const value = cell.getValue<boolean | null>();
+            return value ? (
+              <CircleIcon sx={{ transform: 'scale(0.5)' }} />
+            ) : null;
+          },
+        };
```

Comment:
You think this UI should be the default for all Booleans? Is that how it is currently?

Reply:
Yeah we have used `DotCell` for most booleans 👀